### ISSUE TITLE
Two login options not responding

### ISSUE THREAD

**Issue by ahmedfakhery** [tailscale/tailscale#9481]
### What is the issue?

/

### Steps to reproduce

@

https://github.com/tailscale/tailscale/assets/34455894/916ec913-008d-4896-8403-3ac8ae426d38

### OS

Android

### OS version

12.5.8

### Tailscale version

1.48.2

**Comment by DentonGentry** [comment]
Sign in with other means to start a browser. What browser is installed on this device?

Are you familiar with `adb`? The app isn't getting to the point where it would begin sending telemetry, we may need to see what adb says when the two buttons are clicked.

**Comment by ahmedfakhery** [comment]
> Sign in with other means to start a browser. What browser is installed on this device?
> 
> Are you familiar with `adb`? The app isn't getting to the point where it would begin sending telemetry, we may need to see what adb says when the two buttons are clicked.

No, I am not familiar with adb.
I used Chrome and it works fine before when I reauthenticate. I re-install the app and tried to clear the cache from the Chrome but still does not work.

**Comment by arunsathiya** [comment]
>I used Chrome and it works fine before when I reauthenticate. I re-install the app and tried to clear the cache from the Chrome but still does not work.

Is Chrome also marked as your default browser? Also, are you able to mark a different browser like Firefox as your default to test?

**Comment by ahmedfakhery** [comment]
> >I used Chrome and it works fine before when I reauthenticate. I re-install the app and tried to clear the cache from the Chrome but still does not work.
> 
> Is Chrome also marked as your default browser? Also, are you able to mark a different browser like Firefox as your default to test?

Yes, it is my default browser and I change my default browser to Mi Browser then I use Firefox and I still have the same issue.

Also, the problem is not in my smartphone only but I face this problem with many others.

**Comment by vvirtues** [comment]
for some reason my secure folder works though
tailscale 1.48.2
